deploy/helm/sumologic/values.yaml

```diff
@@ -597,10 +603,10 @@ otelcol:
             #serviceName: service
             statefulSetName: statefulset
           annotations:
-            - tag_name: pod_annotation_%s
+            - tag_name: "annotation:%s"
               key: "*"
           labels:
-            - tag_name: pod_label_%s
+            - tag_name: "label:%s"
```

Comment:
we starting to be inconsistent with the rest of the solution

Reply:
Or it's the opposite, e.g. https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/fluent-plugin-kubernetes-sumologic/lib/fluent/plugin/filter_kubernetes_sumologic.rb#L52

I am going to prepare a summary later and run it by the team. There are several alternative naming solutions being used across the code

Reply:
Changed it back to `pod_labels_%s`